dist/logs/package-lock.json

```diff
@@ -1,15 +1,209 @@
 {
-  "name": "logzio-logs-function",
```

Comment:
please update the following packages to the latest possible version ( see errors of Dependabot )
- yargs-parser
- lodash
- minimist
- kind-of